### PR TITLE
Womanium Final Assignment-Helarie Rose Medie Fah

### PR DESCRIPTION
**Pull request by helarie98**
### PR Description

Pull Request for Quantum Walk algorithm

### CODE REVIEW

File: README_resources/Womanium Final Assignment Helarie Rose Medie Fah [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (helarie98) [inline README_resources/Womanium Final Assignment Helarie Rose Medie Fah:1]: [Helarie_hw4.zip](https://github.com/user-attachments/files/16273891/Helarie_hw4.zip)

Comment (helarie98) [inline README_resources/Womanium Final Assignment Helarie Rose Medie Fah:1]: [Helarie_Fah_hw4.zip](https://github.com/user-attachments/files/16273959/Helarie_Fah_hw4.zip)

**Comment by Nadav138** [comment]
Hello @helarie98 !

Please read the instructions on submission again and resubmit using the correct format and location ([here](https://github.com/Classiq/classiq-library/tree/main/community/womanium/assignments)).
Thanks!

**Comment by helarie98** [comment]
Dear Nadav

I have send it. Please see the link to the github:
https://app.reviewnb.com/Classiq/classiq-library/pull/402/files/

regards,
Helarie

**Pull request closed without merging**